community/sysdig/templates/daemonset.yaml

```diff
@@ -0,0 +1,126 @@
+{{- if .Values.sysdig.accessKey }}
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: {{ template "sysdig.fullname" . }}
+  labels:
+    app: {{ template "sysdig.fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  template:
+    metadata:
+      name: {{ template "sysdig.fullname" .}}
+      labels:
+        app: {{ template "sysdig.fullname" .}}
+        role: monitoring
+    spec:
+      serviceAccountName: {{ template "sysdig.serviceAccountName" .}}
+      tolerations:
+{{ toYaml .Values.tolerations | indent 8 }}
+      hostNetwork: true
+      dnsPolicy: ClusterFirstWithHostNet
+      hostPID: true
+      terminationGracePeriodSeconds: 5
+      {{- if .Values.image.pullSecrets }}
+      imagePullSecrets:
+{{ toYaml .Values.image.pullSecrets | indent 8 }}
+      {{- end }}
+      containers:
+        - name: {{ .Chart.Name }}
+          image: {{ template "sysdig.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          resources:
+{{ toYaml .Values.resources | indent 12 }}
+          securityContext:
+            privileged: true
+          {{- if .Values.ebpf.enabled }}
+          env:
+            - name: SYSDIG_BPF_PROBE
+              value:
+          {{- end }}
+          readinessProbe:
+            exec:
+              command: [ "test", "-e", "/opt/draios/logs/running" ]
+            initialDelaySeconds: 10
```

Comment:
We found that this probe failed four times before coming up; you might consider bumping this a bit so that doesn't happen. Not mandatory.

Reply:
Yes, perhaps only 10 seconds is not long enough to fit the time of kernel module compiling / downloading phase. Meanwhile, we can continue with that value and I'm going to try to get more knowledge about the 10 seconds, because perhaps we can increase it a bit more.

Thanks!